tests/test_hilbert_sort.py

```diff
@@ -24,7 +24,7 @@ def get_max_block_distances(coords: NDArray, box: NDArray, block_size: int) -> N
         start_idx = bidx * block_size
         end_idx = min((bidx + 1) * block_size, N)
         block = coords[start_idx:end_idx]
-        block_distances.append(np.max(delta_r(block[:, None], block[None, :], box=box)))
+        block_distances.append(np.max(np.linalg.norm(delta_r(block[:, None], block[None, :], box=box), axis=-1)))
```

Comment:
+1, good fix, didn't realize it was also in this file

Reply:
This still looks fishy to me -- `np.linalg.norm` without an axis specified returns a scalar, so it doesn't seem like `np.max` is doing anything? I might be confused, but seems like it would be useful to annotate some of the array shapes with comments.

Reply:
Never mind, sorry. I misread the parentheses. Looks good to me.